app/services/ContributionsFrontendService.scala

```diff
@@ -0,0 +1,92 @@
+package services
+
+import cats.data.EitherT
+import play.api.libs.ws.{WSClient, WSRequest, WSResponse}
+import play.api.mvc.RequestHeader
+import play.api.libs.json.{Json, Reads, Writes}
+
+import scala.concurrent.duration._
+import cats.implicits._
+import ophan.thrift.componentEvent.ComponentType
+import ophan.thrift.event.{AbTest, AcquisitionSource}
+import play.api.Logger
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object ContributionsFrontendService {
+  def apply()(implicit wsClient: WSClient): ContributionsFrontendService = new ContributionsFrontendService(wsClient)
+}
+class ContributionsFrontendService(wsClient: WSClient) {
+  import IdentityServiceEnrichers._
+
+  val apiUrl = "https://contribute.thegulocal.com/"
+
+  private def headers(request: RequestHeader): List[(String, String)] = List("Content-Type" -> "application/json")
+
+  private def get[A](
+    endpoint: String,
+    headers: List[(String, String)],
```

Comment:
It is confusing that you have a `def` called headers and a parameter here called headers